packages/playground-examples/copy/pt/Typescript/Language/Structural Typing.ts

```diff
@@ -0,0 +1,84 @@
+// TypeScript é um Sistema de Tipagem Estrutural, isso significa
+// que comparando tipos, o Typescript leva em consideração apenas
+// as propriedades do tipo.
+
+// Isso em contraste a sistemas de tipagem nominais, onde você
+// pode criar dois tipos mas não pode atribuir um ao outro.
+// Veja o exemplo: nominal-typing
```

Comment:
```suggestion
// Veja example:nominal-typing
```

Reply:
thanks!

Reply:
> Great job!
> We are translating some code in the examples as well, to greatly reduce the barrier of entry for Portuguese-only speakers.
> 
> You could translate the code to use `Bola` instead of `Ball`, for instance. However, I would happily approve the PR with the code in English, as the rest is is great shape!

I also thought that having the documentation in portuguese and code in english would make things a little bit harder for the newcomers. I haven't translated the code before because I saw other merged PRs that didn't do that, I was thinking that it would be some kind of pattern.

So I'll translate the code as well. Thank you for the review.

Reply:
I also thought that having the documentation in portuguese and code in english would make things a little bit harder for the newcomers. I haven't translated the code before because I saw other merged PRs that didn't do that, I was thinking that it would be some kind of pattern.

So I'll translate the code as well. Thank you for the review @danilofuchs .